main.tf

```diff
@@ -43,7 +43,7 @@ resource "google_project_service" "cloudsql_api" {
 
 module "google_postgres_db" {
   source                          = "GoogleCloudPlatform/sql-db/google//modules/postgresql"
-  version                         = "4.5.0"
+  version                         = "5.0.0"
```

Comment:
How did we test this PR https://github.com/airasia/terraform-google-postgres_db/pull/6 with 4.5 then?